confiture-web-app/src/components/ui/FileUpload.vue

```diff
@@ -120,36 +120,34 @@ function onFileRequestFinished() {
           class="fr-label fr-upload-group__desc"
           :class="{ 'fr-text--bold': boldTitle }"
         >
-          {{ title }}<br />
-          <span class="fr-mt-1v fr-text--regular fr-hint-text"
-            ><span>Taille maximale par fichier&#8239;: {{ maxFileSize }}</span
-            ><span>. <span v-html="acceptedFormatsHtml"></span></span>
-            <span v-if="multiple">. Plusieurs fichiers possibles.</span></span
-          >
+          {{ title }}
+        </p>
+        <p class="fr-text--regular fr-hint-text fr-my-2v">
+          <span>Taille maximale par fichier&#8239;: {{ maxFileSize }}</span
```

Comment:
On peut utiliser une balise `<template>` non ?

Reply:
J’imagine que tu parles du [_built-in element_ de Vue](https://vuejs.org/api/built-in-special-elements#template) ?
Je ne vois pas trop où l’utiliser ici ?